instagram/src/CommentInput.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+const CommentInput = props => {
+  return (
+    <form onSubmit={this.addComment}>
```

Comment:
Line 5: It should be `props.addComment`

Line 6: should be `onChange={props.commentHolder}`